### ISSUE TITLE
Test failure in opm-simulatores when used together with dune 2.5 

### ISSUE THREAD

**Issue by totto82** [OPM/opm-simulators#1455]
> compareECLFiles_flow+2D_THREEPHASE_POLY_HETER (Failed)

The 2d polymer test doesn't converge when I use it together with dune 2.5. It works fine with dune 2.4. 

```
Time step 1, stepsize 6.67156e-05 days.
Exception occured on process 0 during setup of ILU0 preconditioner with message: MatrixBlockError [bilu0_decomposition:/home/thsa/workspace/opm-release1804/dune-istl/dune/istl/ilu.hh:89]: ILU failed to invert matrix block A[86][86]FMatrixError [luDecomposition:/home/thsa/workspace/opm-release1804/dune-common/dune/common/densematrix.hh:824]: matrix is singular
Exception occured on process 0 during setup of ILU0 preconditioner with message: MatrixBlockError [bilu0_decomposition:/home/thsa/workspace/opm-release1804/dune-istl/dune/istl/ilu.hh:89]: ILU failed to invert matrix block A[86][86]FMatrixError [luDecomposition:/home/thsa/workspace/opm-release1804/dune-common/dune/common/densematrix.hh:824]: matrix is singular

Error: Solver failed to converge after cutting timestep 10 times.

Error: Program threw an exception: [/home/thsa/workspace/opm-release1804/opm-simulators/opm/simulators/timestepping/AdaptiveTimeStepping_impl.hpp:356] Solver failed to converge after cutting timestep 10 times.
```
Didn't we use to do the inversion ourself, in the invertMatrix() methods in ISTLSovler.hpp?

What dune version do we ship with the release?

**Comment by andlaus** [comment]
Dune 2.4 is currently  probably the most important version as it is shipped with ubuntu 16.04. In the near future, Dune 2.5 will likely replace it because that's what's shipped by the latest stable debian release as well as Ubuntu 18.04.

**Comment by blattms** [comment]
My emal reply did not show up within half an hour. Here it is:

@totto82 fixed this with #1464

Closing.
